Platform/Teams/Triage/Rules of Engagement with Triage.md

```diff
@@ -0,0 +1,35 @@
+# Preface
+This document will help all team members working on VA.gov understand when it is appropriate to engage the Triage team for support and provide relevant information to assist in routing reported issues to other appropriate teams.  Because of the limited number of resources on Triage, they can not be a "catch-all" for every issue reported on the platform and every application.  Please read the following document carefully in order to determine the appropriate path for you incident.
+
+## When it is OK to engage the Triage Team
+
+* When the incident is believed to be **Severity 1** which can be defined as
+  * A significant amount of users are impacted
+  * Issue resolution is required in a 24 hour period
+  * Part of (or all of) va.gov is severely disrupted
```

Comment:
I wonder if we can dredge up concrete examples here, for what is "severely disrupted," so we don't have to leave that up to interpretation.